doc/_static/custom.css

```
@@ -25,6 +25,11 @@ table.docutils td li {
 
 }
 
+table td.vl-type-def {
+  max-width: 200px;
+  overflow-x: hidden;
```

```suggestion
  overflow-x: clip;
```

For me, "hidden" shows overlapping text.